src/fides/data/sample_project/privacy_center/config/examples/basic.json

```diff
@@ -0,0 +1,36 @@
+{
+  "title": "Privacy center",
```

Comment:
This looks wrong - no need to include all these in the sample privacy center config. Can you revert all the changes to the `sample_project` folder, basically?

Reply:
yup, not sure what happened here